server/routes/api/helpscout.ts

```diff
@@ -36,6 +44,44 @@ function safe(text: string) {
 const EmailNotFoundTemplate = new Template<IHelpScoutEmailNotFoundTemplate>("helpscout/email_not_found.html");
 const MainHelpScoutTemplate = new Template<IHelpScoutMainTemplate>("helpscout/main.html");
 
+async function getFormAnswers(userData: IFormItem[], branch: string): Promise<IHelpScoutFormItem[]> {
+	let branchName = await Branches.BranchConfig.getCanonicalName(branch);
+	let questionBranch = branchName ? await Branches.BranchConfig.loadBranchFromDB(branchName) : null;
+	if (questionBranch) {
+		const hsQuestionNames = questionBranch?.questions
+			.filter(question => question.showInHelpScout)
+			.map(question => question.name);
+
+		return userData
+			.filter(question => hsQuestionNames.includes(question.name))
+			.map((question: IFormItem): IHelpScoutFormItem => {
+				let name = question.name.replace("-", " ");
```

Comment:
If you want to replace more than just the first occurrence of `"-"` here, use `/-/g` as the pattern instead

Reply:
Good catch